GitPrompt.ps1

```diff
@@ -306,8 +306,4 @@ $PoshGitVcsPrompt = {
     Write-GitStatus $GitStatus
 }
 
-# Install handler for removal/unload of the module
 $Global:VcsPromptStatuses += $PoshGitVcsPrompt
-$ExecutionContext.SessionState.Module.OnRemove = {
-    $Global:VcsPromptStatuses = $Global:VcsPromptStatuses | Where-Object { $_ -ne $PoshGitVcsPrompt }

```

Comment:
And as a "module level" operation, posh-git.psm1 seemed like a more fitting place for this handler.